src/main/java/com/parpiiev/time/services/interfaces/ActivityRequestService.java

```diff
@@ -0,0 +1,24 @@
+package com.parpiiev.time.services.interfaces;
+
+import com.parpiiev.time.services.BaseService;
+import com.parpiiev.time.utils.dto.ActivityRequestDTO;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface ActivityRequestService<T> extends BaseService<T> {
+
+    Optional<T> getByUserIdActivityId(int userId, int activityId);
```

Comment:
Please don't use SpringData naming convention on the Service layer, you can be more verbose to simplify reading :) 
```suggestion
    Optional<T> getByUserIdAndActivityId(int userId, int activityId);
```